vpr/src/place/move_transactions.h

```diff
@@ -48,17 +48,23 @@ struct t_pl_blocks_to_be_moved {
     t_pl_blocks_to_be_moved(const t_pl_blocks_to_be_moved&) = delete;
     t_pl_blocks_to_be_moved(t_pl_blocks_to_be_moved&&) = delete;
 
-/**
- * @brief This function increments the size of the moved_blocks vector and return the index
- * of the newly added last elements. 
- */
+    /**
```

Comment:
These three functions are `BlkLocRegistry` modifiers. Is moving them to `BlkLocRegistry` class a good idea?